src/patternfly/components/Label/styles.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../patternfly-utilities";
+
+.pf-c-label {
+  // Component
+  --pf-c-label--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-label--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-label--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-label--BorderRadius: var(--pf-global--BorderRadius--sm);
+
+  // Color
+  --pf-c-label--BackgroundColor: var(--pf-global--primary-color--100);
+  --pf-c-label--Color: var(--pf-global--Color--light-100);
+
+  // Font
+  --pf-c-label--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-label--FontWeight: var(--pf-global--FontWeight--regular);
+
+  // Modifiers
+  --pf-c-label--hover--BackgroundColor: var(--pf-global--primary-color--200);
+  --pf-c-label--m-compact--FontSize: var(--pf-global--FontSize--xs);
+
+  display: inline-block;
+  padding:
+    var(--pf-c-label--PaddingTop)
+    var(--pf-c-label--PaddingRight)
+    var(--pf-c-label--PaddingBottom)
+    var(--pf-c-label--PaddingLeft);
+  font-size: var(--pf-c-label--FontSize);
+  font-weight: var(--pf-c-label--FontWeight);
+  line-height: var(--pf-c-label--LineHeight);
+  color: var(--pf-c-label--Color);
+  text-align: center;
```

Comment:
doesn't look like this is necessary

Reply:
I wondered that too, I didn't comment though b/c I've seen it done elsewhere like this. I agree though, we should take it out if there's no need for it.

Reply:
okay

Reply:
did you mean to remove this, or want to keep it?